Sources/ProjectSpec/Dependency.swift

```diff
@@ -137,6 +144,7 @@ extension Dependency: JSONEncodable {
             "embed": embed,
             "codeSign": codeSign,
             "link": link,
+            "conditionalPlatforms": conditionalPlatforms.map(Array.init)?.map(\.rawValue)
```

Comment:
Can we add a sort to make this deterministic. And I also don't think the Array init is required